tensordict/tensordict.py

```diff
@@ -1156,6 +1156,74 @@ def memmap_(self, prefix=None, copy_existing=False) -> TensorDictBase:
         """
         raise NotImplementedError(f"{self.__class__.__name__}")
 
+    def memmap_like(self, prefix=None) -> TensorDictBase:
```

Comment:
There's quite a bit of shared logic here with `memmap_`. I wonder if there's some way to reuse.

I guess a dumb version of this function would be something like

```python
return self.clone().memmap_(...)
```

But that results in unnecessary copies.

Reply:
no not in the case of `expand` above
`clone()` will allocate the memory on device / ram! That's a disaster if you have a tensor that is small but expanded millions of times